.github/workflows/ci.yml

```diff
@@ -1,56 +1,77 @@
 name: CI
 
-# These trigger patterns courtesy of https://github.com/broccolijs/broccoli/pull/436
 on:
-  pull_request:
   push:
-    # filtering branches here prevents duplicate builds from pull_request and push
     branches:
-      - master
-      - 'v*'
-    # always run CI for tags
-    tags:
-      - '*'
+      - main
+  pull_request: {}
 
-env:
-  CI: true
+concurrency:
+  group: ci-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
 
 jobs:
-  test-locked-deps:
-    name: Locked Deps
+  test:
+    name: 'Tests'
     runs-on: ubuntu-latest
+
     steps:
-      - name: Checkout Code
-        uses: actions/checkout@v1
+      - uses: actions/checkout@v3
       - name: Install Node
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v3
         with:
-          node-version: 10
+          node-version-file: 'package.json'
+          cache: yarn
       - name: Install Dependencies
         run: yarn install --frozen-lockfile
       - name: Lint
-        run: yarn lint
-      - name: Test
-        run: yarn test:ember
+        run: yarn run lint
+      - name: Run Tests
+        run: yarn run test
```

Comment:
[nit] There's some inconsistency in your invocation of the test command here and below